src/common/Types.ts

```diff
@@ -350,16 +350,10 @@ export type ComponentBase = React.Component<any, any>;
  * Components
  */
 
-// Redefine RefObject here rather than using React.RefObject because
-// older versions of React don't support it.
-interface RefObject<T> {
-    readonly current: T | null;
-}
-
-export interface CommonProps {
+// Rely on RefAttributes type, which exists only in newer typescript definitions, but has support from React 16.3 (released Mar 29, 2018)
+export interface CommonProps<C = React.Component> extends React.RefAttributes<C> {
```

Comment:
I don't think this will. Refs have been supported since 16.3, ReactXP has React@16.5 as a minimum version, so there shouldn't be any breaking change here unless a consumer manually specifies a version below what ReactXP package.json suggests it supports